metatensor-torch/src/register.cpp

```diff
@@ -215,12 +268,28 @@ TORCH_LIBRARY(metatensor, m) {
         ;
 
     // standalone functions
-    m.def("version", metatensor_torch::version);
+    m.def("version() -> str", metatensor_torch::version);
+
+    m.def(
+        "load(str path) -> __torch__.torch.classes.metatensor.TensorMap",
+        metatensor_torch::load
+    );
+    m.def(
+        "load_buffer(Tensor buffer) -> __torch__.torch.classes.metatensor.TensorMap",
+        metatensor_torch::load_buffer
+    );
+
+    m.def(
+        "load_labels(str path) -> __torch__.torch.classes.metatensor.Labels",
+        metatensor_torch::load_labels
+    );
+    m.def(
+        "load_labels_buffer(Tensor buffer) -> __torch__.torch.classes.metatensor.Labels",
```

Comment:
This looks weird, this function is covered by the metatensor-core Python API with the `load_labels`, so why don't we cover it here too by `load_labels`? Same as we do it for `save_buffer`.

Reply:
not sure what you mean? This is the same function as the one in `-core`, but registered as a custom TorchScript operation (so models can use it). And we have both `load_labels` and `load_labels_buffer` in both cases.

Reply:
On the doc preview I don see `load_labels_buffer` in the Python API, maybe it is then just missing in the autofunction .rst? 
https://metatensor--455.org.readthedocs.build/en/455/core/reference/python/io.html#metatensor.load_labels

Reply:
Oh, right. There is indeed a different API here, `load_labels_buffer` is accessible by giving a file-like object (which can be a buffer) to `load_labels`. We could add a more explicit `load_labels_buffer(buffer: bytes | bytearray | memoryview)` function to the -core API.

I'll see if it is simple enough to include in this PR!

Reply:
done